addons/hr_skills_slides/models/slide_channel.py

```diff
@@ -9,19 +9,21 @@ class SlideChannelPartner(models.Model):
 
     def _recompute_completion(self):
         res = super(SlideChannelPartner, self)._recompute_completion()
-        partner_has_completed = {channel_partner.partner_id.id: channel_partner.channel_id for channel_partner in self}
-        employees = self.env['hr.employee'].sudo().search([('user_id.partner_id', 'in', list(partner_has_completed.keys()))])
-        for employee in employees:
-            line_type = self.env.ref('hr_skills_slides.resume_type_training', raise_if_not_found=False)
-            channel = partner_has_completed[employee.user_id.partner_id.id]
-            self.env['hr.resume.line'].create({
-                'employee_id': employee.id,
-                'name': channel.name,
-                'date_start': fields.Date.today(),
-                'date_end': fields.Date.today(),
-                'description': channel.description,
-                'line_type_id': line_type and line_type.id,
-                'display_type': 'course',
-                'channel_id': channel.id
-            })
+        partner_has_completed = {channel_partner.partner_id.id: channel_partner.channel_id
+                                 for channel_partner in self if channel_partner.completed}
+        if partner_has_completed:
```

Comment:
i only add these 2 line to prevent duplicate on hr.resume.line (create resume line only if a partner completed a channel)